CDS/src/org/icpc/tools/cds/service/ContestRESTService.java

```diff
@@ -784,6 +793,77 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
 		pw.append("\"" + sId + "\"");
 	}
 
+	protected void handleClarification(HttpServletRequest request, HttpServletResponse response, ConfiguredContest cc,
+			JsonObject obj) throws IOException {
+		String id = obj.getString("id");
+		String time = obj.getString("time");
+		String fromTeamId = obj.getString("from_team_id");
```

Comment:
No to_team_id support? (for admins)

Reply:
The JSON object is passed through, so admin request with to_team_id is just passed along. I realized I wasn't stopping teams from sending clars to another team though, so added a commit to block that.